examples/CampaignManagement/AddCompleteCampaignsUsingMutateJob.php

```diff
@@ -164,22 +200,52 @@ public static function runExample(GoogleAdsClient $googleAdsClient, $customerId)
             $response->getNextSequenceToken(),
             PHP_EOL
         );
+    }
 
-        // Runs the mutate job for executing the added operations.
+    /**
+     * Requests the API to run the mutate job for executing all uploaded mutate job operations.
+     *
+     * @param MutateJobServiceClient $mutateJobServiceClient the mutate job service client
+     * @param string $mutateJobResourceName the resource name of mutate job to be run
+     * @return OperationResponse the operation response from running mutate job
+     */
+    private static function runMutateJob(
+        MutateJobServiceClient $mutateJobServiceClient,
+        string $mutateJobResourceName
+    ) {
         $operationResponse = $mutateJobServiceClient->runMutateJob($mutateJobResourceName);
         printf(
             "Mutate job with resource name '%s' has been executed.%s",
             $mutateJobResourceName,
             PHP_EOL
         );
+        return $operationResponse;
+    }
 
-        // Polls the server until the mutate job execution finishes by setting the initial poll
-        // delay time and the total time to wait before time-out.
+    /**
+     * Polls the server until the mutate job execution finishes by setting the initial poll
+     * delay time and the total time to wait before time-out.
+     *
+     * @param OperationResponse $operationResponse the operation response used to poll the server
+     */
+    private static function pollMutateJob(OperationResponse $operationResponse)
+    {
         $operationResponse->pollUntilComplete([
             'initialPollDelayMillis' => self::POLL_FREQUENCY_SECONDS * 1000,
             'totalPollTimeoutMillis' => self::MAX_TOTAL_POLL_INTERVAL_SECONDS * 1000
         ]);
+    }
 
+    /**
+     * Prints all the results from running the mutate job.
+     *
+     * @param MutateJobServiceClient $mutateJobServiceClient the mutate job service client
+     * @param string $mutateJobResourceName the resource name of mutate job to get its results
+     */
+    private static function printResults(
```

Comment:
nit: I would name this function differently to highlight that it requires another API request like "fetchAndPrintResults"